frontend/src/organizations/components/UserOrganizationsList.tsx

```diff
@@ -0,0 +1,42 @@
+import { AxiosError } from 'axios';
+import { Flex, Button, Spinner, Text, Badge } from '@chakra-ui/react';
+import { Link } from 'react-router-dom';
+import { UserOrganizationsListElement } from '@organizations/components/UserOrganizationsListElement';
+import { useGetUserOrganizations } from '@organizations/hooks/api';
+import { ApplicationErrorDto } from '@shared/dtos/application-error';
+import { applicationErrorHandler } from '@shared/helpers/application-error';
+import { reverse } from '@shared/helpers/routing';
+import { useUser } from '@users/contexts';
+
+export const UserOrganizationsList: React.FC = () => {
+  const { isLoggedIn } = useUser();
+
+  const handleError = (error: AxiosError<ApplicationErrorDto<any, any>, unknown>): void => {
+    applicationErrorHandler().handleAxiosError(error);
+  };
+  const { isLoading, data } = useGetUserOrganizations(handleError);
+
+  if (!isLoggedIn) {
```

Comment:
To trzeba wywalic, i dodac security dla routow. Tzn jesli user jest zalogowany i wejdzie na /login lub /register, powinien byc przekierowany na homepage. Natomiast jak user nie jest zalogowany i wchodzi na organizacje, to powinien byc przekierowany na homepage